app/views/layouts/_nav.haml

```diff
@@ -0,0 +1,30 @@
+%nav.navbar.navbar-default.nav-custom
+  .container-fluid
+    .navbar-header
+      %button.navbar-toggle.collapsed{ type: 'button',
+       data: { toggle: 'collapse', target: '#bs-example-navbar-collapse-1' },
+        aria: { expanded: 'false' } }
+
```

Comment:
I don't think haml lint cares. I mention it more as a point of consistency with the rest of the HAML in this project. I typically don't add newlines in HAML. Personally I find that the indentation provides enough of a visual indication of where sections of the code begin and end.